src/main/java/com/ibm/eventstreams/kafkaconnect/plugins/xml/engines/StructToXmlBytes.java

```diff
@@ -94,38 +98,31 @@ private Document createXmlDoc(Schema schema, Object value, String rootName) {
                 for (final Field field : schema.fields()) {
                     processSchema(doc, schemaSequence, field.name(), field.schema());
                 }
-            }
-            else if (schema.type() == Type.ARRAY) {
+            } else if (schema.type() == Type.ARRAY) {
```

Comment:
```suggestion
            } 
            else if (schema.type() == Type.ARRAY) {
```

Unrelated to this specific PR